app/uk/gov/hmrc/exports/services/reversemapping/declaration/locations/LocationsParser.scala

```diff
@@ -57,11 +72,15 @@ class LocationsParser @Inject()(
         officeOfExit = officeOfExit,
         supervisingCustomsOffice = supervisingCustomsOffice,
         warehouseIdentification = warehouseIdentification,
+        inlandOrBorder = inlandOrBorder,
         inlandModeOfTransportCode = inlandModeOfTransportCode
       )
     }
 
   private def parseRoutingCountries(inputXml: NodeSeq)(implicit context: MappingContext): XmlParserResult[Seq[Country]] =
     (inputXml \ Declaration \ Consignment \ Itinerary \ RoutingCountryCode).map(countryParser.parse).toEitherOfList.map(_.flatten)
+}
 
+object LocationsParser {
+  val adtForInlandOrBorder = List(STANDARD_FRONTIER, STANDARD_PRE_LODGED, SUPPLEMENTARY_SIMPLIFIED)
```

Comment:
For 'adt' I read "Abstract Data Type" rather than "Additional Declaration Type" :laughing: Maybe expand on this name for clarity (e.g "additionalDecTypesForInlandOrBorder")?